api/PerformanceTiming.json

```diff
@@ -988,10 +988,10 @@
               "version_added": "32"
             },
             "safari": {
-              "version_added": true
+              "version_added": "10.1"
```

Comment:
This (`toJSON`) is the only outlier in this PR. Indeed it was added later, in https://trac.webkit.org/changeset/205161/webkit, and the timing (Aug 29, 2016 ) is right for 10.1. (It was slightly before 10 was released, but that must have been branched months earlier.)